csv-generator/csv_generator/consumers/consumer.py

```diff
@@ -33,22 +33,42 @@ def output_file(self) -> str:
             self._output_file = os.path.join(self.output_dir, '{0}_{1}'.format(self.create_date, self.base_file_name))
         return self._output_file
 
-    def process(self, element: 'lxml.etree.ElementTree', xml_file_name: str) -> None:
+    def process(self, element: 'lxml.etree.ElementTree', source_file_name: str) -> None:
         """Parse target `lxml.etree.ElementTree` object, extract required data and
         write data row to output_file.
 
         :param element: class: `lxml.etree.ElementTree`
-        :param xml_file_name:
+        :param source_file_name:
         :return:
         """
-        pass
+        raise NotImplementedError()
+
+    # https://github.com/elifesciences/data-pipeline/issues/86
```

Comment:
The comment with the issue reference is just temporary?

Reply:
Weird logic should be justified so I put a link to the ticket. Tickets shouldn't be deleted.